### PR TITLE
Updated changelog to reflect combined 2.2.0 release

### PR DESCRIPTION
**Pull request by holgerd77**
Hi,
just made a short new pull request to reflect the combined release (and with updated date(s)).

Cheers
Holger

### CODE REVIEW

**Comment by holgerd77** [comment]
Could this be merged rather sooner than later, doesn't feel so comfortable with the CHANGELOG in such a confused state and will irritate people...

**Comment by axic** [comment]
@holgerd77 can you please rebase?

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,19 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
-
-
-## [2.3.0] - Unreleased
-### Changed
```
Comment (axic) [inline CHANGELOG.md]: Please keep these two lines, to be filled out as we go.

**Comment by holgerd77** [comment]
Done.

**Comment by axic** [comment]
@holgerd77 sorry merged the buffer change first, this needs a rebase.

**Comment by holgerd77** [comment]
@axic Done again. :-)

**Pull request merged**